### PR TITLE
33resizable columns

### LINKED ISSUE #33: Columns should be resizable
Columns should be resizable

### PR DESCRIPTION
**Pull request by pat310**
closes #33

### CODE REVIEW

File: src/app.jsx [hunk 1]
```diff
@@ -120,6 +120,7 @@ export default class App extends React.Component {
           data={this.state.data}
           selectedAggregationDimension={selectedAggregationDimension}
           colorPack={{
+            columnResizer: 'none',
```
Comment (pat310) [inline src/app.jsx]: @turnerniles Setting default color for the column resizing bars to transparent for now

File: src/components/Pivot/styles.scss [hunk 2]
```diff
@@ -121,72 +121,6 @@ section.virtualized-pivot{
 	  transition: all 0.3s cubic-bezier(.25,.8,.25,1);
 	}
 
-	section.pivot-grid{
```
Comment (pat310) [inline src/components/Pivot/styles.scss]: @turnerniles It seemed like this was all duplicated css from `Table/styles.scss`

**Comment by pat310** [comment]
@turnerniles The `Table` file is starting to get long - would it be cool if I separate it into separate files (not components but just files within the Table directory)?

**Comment by turnerniles** [comment]
@pat310 It would be very cool!

**Pull request merged**